integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -310,14 +310,6 @@ def get_grades_summary(
         else:
             completed_date_from_api, grade_from_api, is_passing_from_api, grade_percent = \
                 self.collect_grades_data(enterprise_enrollment, course_details, channel_name)
-            LOGGER.info(generate_formatted_log(
```

Comment:
added during a previous investigation, noisy and limited value after previous fix.